tests/security/MemberTest.php

```diff
@@ -185,16 +185,44 @@ public function testPasswordChangeLogging() {
 	 * Test that changed passwords will send an email
 	 */
 	public function testChangedPasswordEmaling() {
+		$oldSetting = Config::inst()->get('Member', 'notify_password_change');
+		Config::inst()->update('Member', 'notify_password_change', true);
```

Comment:
You don't need to store the old settings, the most you have to do is nest and unnest the config like so:

```php
function testThing() {
    Config::nest();
    //test code
    Config::unnest();
}
```

But I think we actually made each test "sandboxed" now so the nesting and unnesting is done automatically.